docker-compose.yml

```diff
@@ -1,13 +1,13 @@
 version: "3"
 services:
   minio:
-    image: docker.io/minio/minio:RELEASE.2022-03-26T06-49-28Z
+    image: docker.io/minio/minio:RELEASE.2022-07-30T05-21-40Z
     ports:
       - "9000:9000"
       - "9001:9001"
     environment:
-      MINIO_ACCESS_KEY: minio
-      MINIO_SECRET_KEY: minio123
+      MINIO_ROOT_USER: minio
```

Comment:
I think we'll need to update some other references to "MINIO_ACCESS_KEY" on the code in a follow-up PR. [[1](https://cs.github.com/aminueza/terraform-provider-minio?q=MINIO_ACCESS_KEY)]

Reply:
Oh yeah, that is probably a good idea.  I only saw the warning in the container logs while investigating...

I'll have a look at where those are used in the rest of the code and will update this PR.
Thank you for the review!